src/sharing/SharingDetailsModal.jsx

```diff
@@ -3,7 +3,7 @@ import styles from './share.styl'
 import React, { Component } from 'react'
 import { cozyConnect, fetchSharings } from 'cozy-client'
 import { UserAvatar } from './components/Recipient'
-import Modal from 'cozy-ui/react/Modal'
+import Modal, { ModalContent } from 'cozy-ui/react/Modal'
 
 const Owner = UserAvatar
```

Comment:
Not good, we shall have

```
import { UserAvatar as Owner } from './components/Recipient'
```

⚠️ remember that object are mutable in JS:

```js
const elephant = {
 type: 'elephant'
}
const mouse = elephant
mouse.type = 'mouse'

console.log(elephant) // { type: 'mouse' }
console.log(mouse) // { type: 'mouse' }
```
So it's confusing to see `const Owner = UserAvatar`.

Reply:
You are correct and I'm changing it, but it's not my code :-)